core/templates/dev/head/editor/EditorServices.js

```diff
@@ -340,13 +340,12 @@ oppia.factory('changeListService', [
      * is correctly formed
      *
      * @param {object} gadgetData The dict containing new gadget information.
-     * @param {string} panelName The panel where the gadget is being added.
      */
-    addGadget: function(gadgetData, panelName) {
+    addGadget: function(gadgetData) {
       this._addChange({
         cmd: CMD_ADD_GADGET,
         gadget_dict: gadgetData,
-        panel_name: panelName
+        panel_name: gadgetData['panel']
```

Comment:
Oh, okay. I'll make one more commit using .panel consistently across front-end. I like keeping all the panel_name variables on the backend as the simpler 'panel'. I reviewed that the shorter name always makes sense in context. It additionally sets up more intuitive naming for the future notion of gadgets having a 'draggable' panel, instead of a 'draggable' panel_name.